script/install-dev.sh

```diff
@@ -4,6 +4,9 @@ set -e
 # Build app and api containers
 docker-compose -f docker/docker-compose.dev.yml build
 
+# Install dependencies for app
+(cd app && yarn)
+
```

Comment:
Might be able to do this in `app/Dockerfile.dev` instead, which is probably a better place for it.